### ISSUE TITLE
Cancel Button on loginUI

### ISSUE THREAD

**Issue by rejji** [aws-amplify/docs#5852]

**Comment by josefaidt** [comment]
Hey @rejji what issue are you experiencing with the cancel button on the Login UI? It looks like the issue description may have been unintentionally edited and removed

**Comment by rejji** [comment]
Hello @josefaidt I have moved past this and implemented cancel button on the Login UI. [Customization](https://ui.docs.amplify.aws/react/connected-components/authenticator/customization) can include cancel button. A button and a custom function would do good on doc for such purpose. If there is certain functionality that one wants to put on the login ui, I believe it is doable - say, click a button on login UI to chose your avatar. This will serve as a way to allow simple extension of login ui. Also, [amplify doc](https://docs.amplify.aws/start/getting-started/auth/q/integration/react/#bonus-use-amplify-ui-primitives) can have undo/change feedback submitted for "was this page helpful" as one might be checking the behaviour for any post submit action. Delete of github issue is not available, hence I marked it close. Thanks for reaching out and checking.
